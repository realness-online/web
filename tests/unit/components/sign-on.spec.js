import { shallow } from 'vue-test-utils'
import sign_on from '@/components/sign-on'
describe('@/components/sign-on', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(sign_on)
  })
  it('Renders sign on button', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
