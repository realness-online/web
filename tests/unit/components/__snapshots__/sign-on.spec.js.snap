// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@/components/sign-on Renders sign on button 1`] = `
<button
  autofocus="autofocus"
  class="sign-on"
>
  Sign on
</button>
`;
